docs/src/pages/guides/migration-v4/migration-v4.md

```diff
@@ -109,12 +124,14 @@ yarn add @emotion/react @emotion/styled
 
 If you are using `@material-ui/pickers`, it has moved to `@mui/lab`. You can follow [these steps](#material-ui-pickers).
 
-You should have installed `@mui/styles` now.
+You should have installed `@mui/styles` by now.
 It includes JSS, which duplicate with emotion.
 It's meant to allow a gradual migration to v5.
 You should be able to remove the dependency following [these steps](#migrate-from-jss).
 
-After the upgrade and migration of the packages, you can remove the old `@material-ui/*` packages by running `yarn remove` or `npm uninstall`.
+> 📝 Please make sure that your application is still **running** without errors and **commit** the change before continuing the next step.
+
```

Comment:
Because v5 packages are separated from v4, the project should not crash after the installation. Add this note to remind developer to commit before running the codemod.